### DISCUSSION TITLE
Open Preview appears twice when adding the "vscode-builtin-markdown" plugins

### DISCUSSION THREAD

**Discussion by lmcbout** [eclipse-theia/theia#7783] [General] [answered]
Open Preview appears twice: 
- under the context menu 
- under  "Open With" sub-menu

Note: Select any README.md file under Theia

This comes from the following built-in plugins:

- vscode-builtin-markdown

Do we need to keep this build-in or we just use Theia without it and have only one option under "Open with/Preview" for a markdown file ?

**Comment by vince-fugnitto** [comment, marked as answer]
The issue has been tracked previously: https://github.com/eclipse-theia/theia/issues/6984.
There is no harm in keeping both, once it is fully supported `@theia/preview` will be dropped.
